relay-server/src/services/processor/span/processing.rs

```diff
@@ -180,63 +187,50 @@ pub fn extract_from_event(state: &mut ProcessEnvelopeState<TransactionGroup>) {
         state.managed_envelope.envelope_mut().add_item(item);
     };
 
-    let span_metrics_extraction_enabled = state
-        .project_state
-        .has_feature(Feature::SpanMetricsExtraction);
-    let custom_metrics_enabled = state.project_state.has_feature(Feature::CustomMetrics);
-
     let Some(event) = state.event.value() else {
         return;
     };
 
-    let extract_transaction_span = span_metrics_extraction_enabled
-        || (custom_metrics_enabled && !event._metrics_summary.is_empty());
```

Comment:
I assume this extra condition was here for a reason? I recently looked at this with @iambriccardo, did we want to enable transaction spans for anyone who emits custom metrics?

Anyway, I prefer the new version because it's simpler. Just wanted to double-check.

Reply:
We added this at the time where only the `all-modules` feature flag ingested segments, and metrics summaries were on segments only. This allowed us to bypass the span feature flags and ingest metrics summaries.

Now that we're indexing all spans, for all orgs, it's not needed anymore: if we see metrics summaries, we'll ingest them.

Part of the issue here is, as it was bypassing the span feature flag, it allowed us to ingest spans for AM1 customers, which we don't want.